website/content/docs/v0.9/Guides/troubleshooting-control-plane.md

```diff
@@ -0,0 +1,440 @@
+---
+title: "Troubleshooting Control Plane"
+description: "Troubleshoot control plane failures for running cluster and bootstrap process."
+---
+
+This guide is written as series of topics and detailed answers for each topic.
+It starts with basics of control plane and goes into Talos specifics.
+
+This document mostly applies only to Talos 0.9 control plane based on static pods.
+If Talos was upgraded from version 0.8, it might be still running self-hosted control plane, current status can
+be checked with the command `talosctl get bootstrapstatus`:
+
+```bash
+$ talosctl -n IP get bs
+NODE         NAMESPACE   TYPE              ID              VERSION   SELF HOSTED
+172.20.0.2   runtime     BootstrapStatus   control-plane   1         false
+```
+
+In this guide we assume that Talos client config is available and Talos API access is available.
+Kubernetes client configuration can be pulled from control plane nodes with `talosctl -n IP kubeconfig`
+(this command works before Kubernetes is fully booted).
+
+### What is a control plane node
+
+Talos nodes which have `.machine.type` of `init` and `controlplane` are control plane nodes.
+
+The only difference between `init` and `controlplane` nodes is that `init` node automatically
+bootstraps single-node `etcd` cluster on a first boot if etcd data directory empty.
+Node with type `init` can be replaced with `controlplane` node which is triggered to run etcd bootstrap
+with `talosctl --nodes <IP> boostrap` command.
+
+Use of `init` type nodes is discouraged, as it might lead to split-brain scenario if one node in
+existing cluster is reinstalled while config type is still `init`.
+
+It is critical to make sure only one control plane runs in bootstrap mode (either with node type `init` or
+via boostrap API/`talosctl bootstrap`), as having more than node in boostrap mode leads to split-brain
```

Comment:
```suggestion
via boostrap API/`talosctl bootstrap`), as having more than node in bootstrap mode leads to split-brain
```

Reply:
there's actually a bunch of `boostrap`s to fix. I do this one all the time too :man_facepalming: